fn main()
{
let error_message = "An error occurred!";
eprint!("Error: {}", error_message);
}